packages/style-engine/src/styles/background/index.ts

```diff
@@ -101,18 +88,6 @@ const backgroundSize = {
 			)
 		);
 
-		// If background size is set to contain, but no position is set, default to center.
-		if (
-			_backgroundSize === 'contain' &&
-			_backgroundPosition === undefined
-		) {
-			styleRules.push( {
-				selector: options.selector,
-				key: 'backgroundPosition',
-				value: 'center',
-			} );
-		}
-
```

Comment:
Are these styles still being output anywhere? Similar to the comment above for the default value for `backgroundSize`, I couldn't see where this value is being output in the editor for existing markup that might not have this value set.

Reply:
I hadn't tested that yet, but you're right. 

I'll try to find another place to output feature-specific styles. If I can't, I'll revert these changes and worry about it later.

Maybe by adding an addAttributes hook similar to the other hooks, e.g., 

https://github.com/WordPress/gutenberg/blob/f2bf6be69adbf6e9da69e47d3c0b13d0c81ab157/packages/block-editor/src/hooks/color.js#L445-L445